src/OrchardCore.Modules/OrchardCore.Lucene/Views/LuceneAnalyzerSettings.Edit.cshtml

```diff
@@ -0,0 +1,13 @@
+@using Lucene.Net.Util
+@model LuceneSettingsViewModel
+<fieldset class="form-group" asp-validation-class-for="Version">
+    <label asp-for="Version" class="control-label"><span asp-validation-for="Version"></span></label>
+    <select asp-for="Version" asp-items="Html.GetEnumSelectList<LuceneVersion>()" class="form-control"></select>
+    <span class="hint">@T["A lucene version to use which can be left with default value."]</span>
+</fieldset>
+
+<fieldset class="form-group" asp-validation-class-for="Analyzer">
```

Comment:
Is there a way to list the available ones? Even if we need to add a new options for that each module can register them. This way we could imagine to provide a new module that would add more analyzers, or special language modules could provide analyzers for these languages.